src/source/promises.bs

```diff
@@ -1065,6 +1065,8 @@ namespace promises.internal
 
     ' Log the error if crash logging is enabled
     sub logCrashIfEnabled(error as dynamic)
+        ' Filter out user defined errors
+        if error.number = 40 then return
```

Comment:
Just verifying, it's supposed to return here?

Reply:
Yes. 40 is what you get when you intentionally use the `throw` statement to create your own error.